src/Jackett.Common/Definitions/divteam.yml

```diff
@@ -173,4 +179,3 @@ search:
         img[src="images/4x.png"]: 4
         img[src="images/5x.png"]: 5
         "*": 1
-# xbtit customised
```

Comment:
please leave the engine tags in place.
they are useful when looking for similar indexers to compare or to use as a template for a request to add a new indexer that uses the same search engine.

Reply:
Sorry.
Added.